metabase/metabase
# Make filters sticky in Dashboard page

Fixes #11187

## How to Test

1. Visit dashboards that are tall enough to make the window scrollable.
2. Scroll
3. Filter should now be sticky at the top on scroll.

Try opening sidebars: filter, sharing, etc.

Try adding enough filters that the widget has more than one row of filters.

🎗️This PR is exclusively for regular view mode.
Will follow up with similar PRs serving edit, fullscreen modes if prioritized.

### Demo

![Metabase Scroll 1](https://user-images.githubusercontent.com/380816/128077075-7a4106a4-910f-499a-b991-326d72658748.gif)
